### ISSUE TITLE
can it be run without use icvl or nyu importer?

### ISSUE THREAD

**Issue by luchen828** [moberweger/deep-prior-pp#45]
because the icvl or nyu dataset is too big,so can i run test-realtimepipline.py without the parament of 'di'? i mean Can I change it to not use icvl or nyu dataset?

**Comment by moberweger** [comment]
Yes, you can run the the code using `src/test_realtimepipeline.py` without the datasets. Remove the sequence loading and use [/src/test_realtimepipeline.py#L77](https://github.com/moberweger/deep-prior-pp/blob/11f585f73d2c7957c95db302b770a3b4962c0386/src/test_realtimepipeline.py#L77) instead of the `FileDevice`. In that case, only the intrinsic camera parameters from the importer are used.

**Comment by luchen828** [comment]
thank you very much! By the way,which datasetbe be choosed&nbsp; to train the msra pretrained model?


 
---Original---

**Comment by moberweger** [comment]
the msra pretrained model is trained on the MSRA dataset https://www.dropbox.com/s/bmx2w0zbnyghtp7/cvpr15_MSRAHandGestureDB.zip?dl=0

**Comment by luchen828** [comment]
do you train it how much times?


 
---Original---

**Comment by moberweger** [comment]
All training details can be found in [the training script](https://github.com/moberweger/deep-prior-pp/blob/master/src/main_msra15_posereg_embedding_crossval.py)
